tests/testsuite.src/run_fundamental.at

```diff
@@ -2510,6 +2510,102 @@ AT_CHECK([$COBCRUN_DIRECT ./prog], [0],
 AT_CLEANUP
 
 
+AT_SETUP([MOVE to item with +, -, B, 0, / and ,])
+AT_KEYWORDS([fundamental edited editing])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. prog.
+
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 W-P4    PIC S9          VALUE +4.
+       01 W-M4    PIC S9          VALUE -4.
+       01 W-M     PIC -B-         VALUE SPACES.
+       01 W-P     PIC +B+         VALUE SPACES.
+
+       01 W-P123  PIC S9(3)       VALUE +123.
+       01 W-M123  PIC S9(3)       VALUE -123.
+       01 W-MM    PIC ---B--9     VALUE SPACES.
+       01 W-PP    PIC +++B++9     VALUE SPACES.
+
+       01 W-X     PIC -//00BB,,-  VALUE SPACES.
+
+       PROCEDURE DIVISION.
+           MOVE W-P4 TO W-M.
+           DISPLAY W-M.
+           MOVE W-M4 TO W-M.
+           DISPLAY W-M.
+           MOVE W-P4 TO W-P.
+           DISPLAY W-P.
+           MOVE W-M4 TO W-P.
+           DISPLAY W-P.
+
+           MOVE W-P123 TO W-MM.
+           DISPLAY W-MM.
+           MOVE W-M123 TO W-MM.
+           DISPLAY W-MM.
+           MOVE W-P123 TO W-PP.
+           DISPLAY W-PP.
+           MOVE W-M123 TO W-PP.
+           DISPLAY W-PP.
+
+           MOVE W-P4 TO W-X.
+           DISPLAY W-X.
+           MOVE W-M4 TO W-X.
+           DISPLAY W-X.
+
+           MOVE W-P123 TO W-X.
+           DISPLAY W-X.
+           MOVE W-M123 TO W-X.
+           DISPLAY W-X.
+
+           STOP RUN.
+])
+
+AT_CHECK([$COMPILE prog.cob], [0], [], [])
+AT_CHECK([$COBCRUN_DIRECT ./prog], [0],
+[  4
+ -4
+ +4
+ -4
+    123
+   -123
+   +123
+   -123
+         4
+        -4
+         3
+        -3
+])
+
+AT_CLEANUP
+
+
+AT_SETUP([MOVE to shorter edited item (truncation)])
+AT_KEYWORDS([fundamental edited editing])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. prog.
+
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 W-SRC PIC 999V99 VALUE 123.45.
+       01 W-DST PIC ZZZV.
+
+       PROCEDURE DIVISION.
+           MOVE W-SRC TO W-DST.
```

Comment:
please add case of `W-SRC2 99999V9 VALUE 1234.5.` (truncation to the left)

Note: this _should_ raise

```c
/* A non-space or significant-digit was truncated during MOVE (COBOL2025) */
COB_EXCEPTION  (0308, COB_EC_DATA_TRUNCATION,
		"EC-DATA-TRUNCATION", 0)
```

which we don't have yet and also need to add for the other `MOVE` cases... this can be done with a follow-up, but should be kept in mind (at least add as a `#if 0 /* TODO: add in general` + `cob_set_exception (COB_EC_DATA_TRUNCATION);`. please - then again you may also just add it in exception.def...)

... and if/when you add that, please also add the definition for

```c
/* Attempt to use a null valued data item in a context where not permitted (COBOL2025) */
COB_EXCEPTION  (0307, COB_EC_DATA_NULL,
		"EC-DATA-NULL", 1)
```